source/geod24/LocalRest.d

```diff
@@ -740,7 +740,12 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
 
                         auto command = Command(C.thisTid(), scheduler.getNextResponseId(), ovrld.mangleof,
                                                SerializedData(serialized));
-                        C.send(this.childTid, command);
+
+                        // If the node already shut down, its MessageBox will be
+                        // closed. Detect it and notify the user.
+                        // Note that it might be expected that the remote died.
+                        if (!C.trySend(this.childTid, command))
+                            throw new Exception("Connection with peer closed");
```

Comment:
So it seems like in our case when a node receives its `shutdown` call, we should stop all the timers (the fibers) that are currently running. Like the periodic `getBlocksFrom`.

Reply:
I was originally thinking so, but... Do we really need to ? Those methods should handle client disconnection gracefully anyway.